### PR TITLE
Ports welding overlay from the other server

### PR DESCRIPTION
**Pull request by etherware-novice**
# About The Pull Request
Makes welding sparks appear when welding a wall, ported from Citadel Station (https://github.com/Citadel-Station-13/Citadel-Station-13/pull/15702)

## Why It's Good For The Game
More visual input

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
:add: adds welding sparks
/:cl:

### CODE REVIEW

**Comment by etherware-novice** [comment]
https://github.com/Citadel-Station-13/Citadel-Station-13/pull/15702

**Comment by nednaZ** [comment]
Please put the PR you ported from in the base of the PR.
Also make the changelog an "add" rather than "tweak"

File: code/game/objects/items/tools/weldingtool.dm [hunk 1]
```diff
@@ -109,6 +109,12 @@
 	dyn_explosion(T, plasmaAmount/5)//20 plasma in a standard welder has a 4 power explosion. no breaches, but enough to kill/dismember holder
 	qdel(src)
 
+/obj/item/weldingtool/use_tool(atom/target, mob/living/user, delay, amount, volume, datum/callback/extra_checks, skill_gain_mult)
```
Comment (nednaZ) [inline code/game/objects/items/tools/weldingtool.dm:112]: We don't have 'skill_gain_mult' on this server.

**Review by nednaZ — CHANGES_REQUESTED** [review]
Looks good outside of the extra variable.

**Review by nednaZ — APPROVED** [review]
Looks good to me.

**Pull request merged**